public/webpack.config.js

```diff
@@ -51,4 +118,4 @@ module.exports = {
         "react": "React",
         "react-dom": "ReactDOM",
     },
-};
\ No newline at end of file
+};
```

Comment:
Should there be `;` here? I think our typescript formatter would remove them. But maybe they are necessary for javascript?